src/Rx.KitchenSink.ts

```diff
@@ -8,6 +8,8 @@ import {Scheduler as IScheduler} from './Scheduler';
 export interface KitchenSinkOperators<T> extends CoreOperators<T> {
   isEmpty?: () => Observable<boolean>;
   elementAt?: (index: number, defaultValue?: any) => Observable<T>;
+  distinct?: (compare?: (x: T, y: T) => boolean, flushes?: Observable<any>) => Observable<T>;
```

Comment:
`distinct` seems added in `Rx` module but definition leaves in `KitchenSinkOperator` instead of `CoreOperator`. Which module do we want to have those operator?